app/models/manageiq/providers/azure/event_catcher_mixin.rb

```diff
@@ -14,6 +14,10 @@ def parse_event_action(event_action)
     "#{object_class}_#{event_type}_"
   end
 
+  def parse_provider_type(event)
+    event["resourceProviderName"]["value"]
+  end
+
```

Comment:
@djberg96 - Yea Im on the fence on that one. Might be easier to understand what's being parsed if you don't have to go looking for the method. I'll change it.